backups/cognito/backup_users.py

```diff
@@ -0,0 +1,76 @@
+import boto3
+import csv
+from datetime import datetime
+from botocore.exceptions import ClientError
+import os
+
+client = boto3.client('cognito-idp')
+s3_client = boto3.client('s3')
+
+environment='production'
+user_pool_name = 'biomage-user-pool-case-insensitive-{}'.format(environment)
+backup_bucket = 'biomage-backups-{}'.format(environment)
+
+def backup_users():
+    try:
+        all_users = get_all_users()
+        num_users = len(all_users)
+    except Exception as e:
+        raise Exception('Getting current cognito users has failed ', e)
+
+    csv_headers = ['cognito:username','name','email','email_verified',
```

Comment:
put this as a constant either at the top of the file or another file if there are enough costants